fs/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![cfg(feature = "agave-unstable-api")]
```

Comment:
this guy snuck in in the meantime. probably need to add a ci check. something like
```
if package.publish != false || src/lib.rs.lines(0) != '#![cfg(feature = "agave-unstable-api")]' { exit 1 }```